modules/core/zio/src/weaver/ziocompat/suites.scala

```diff
@@ -42,8 +45,15 @@ abstract class BaseMutableZIOSuite[Res <: Has[_]](implicit tag: Tag[Res])
       }
       if (filteredTests.isEmpty) Stream.empty // no need to allocate resources
       else {
-        val suiteLayer = sharedLayer.passthrough
         for {
+          ref <-
+            Stream.resource(
+              FiberRef.make(Chain.empty[Log.Entry]).toManaged_.toResourceZIO)
+          perTestLayer: RLayer[ZEnv, LogModule with ZEnv] =
```

Comment:
it ain't specific to a test anymore (you've renamed PerTestEnv to Env, you should rename this value too)